srcpkgs/solvespace/template

```diff
@@ -0,0 +1,20 @@
+# Template file for 'solvespace'
+pkgname=solvespace
+version=2.3
+revision=1
+wrksrc=solvespace
+only_for_archs="i686 x86_64 x86_64-musl"
```

Comment:
why these arches? Is this not equivalent to nocross=yes?

Reply:
> It is more a coincidence that these 3 archs

... and i686-musl even though there is no official builder for it.

Reply:
Honestly, I just don't have any platform to test native arm compilation, and it's only a case of some tools it compiles (which appear to just be build helpers?) which end up being cross compiled by cmake. I have no reason to believe it would fail to compile on native arm, but cannot test.

https://travis-ci.org/bobertlo/void-packages/jobs/487251265

Could I link to that? I could try to fix cross compilation but I don't really know cmake at all, and was not able to find a way to do so.